bigframes/ml/utils.py

```diff
@@ -17,33 +17,41 @@
 
 import bigframes_vendored.constants as constants
 from google.cloud import bigquery
+import pandas as pd
 
 from bigframes.core import blocks
 import bigframes.pandas as bpd
 
 # Internal type alias
-ArrayType = Union[bpd.DataFrame, bpd.Series]
+InputArrayType = Union[bpd.DataFrame, bpd.Series, pd.DataFrame, pd.Series]
+OutputArrayType = Union[bpd.DataFrame, bpd.Series]
 
 
-def convert_to_dataframe(*input: ArrayType) -> Generator[bpd.DataFrame, None, None]:
+def convert_to_dataframe(
```

Comment:
Do you think we can merge the logic in this file into the core/convert module logic? Ideally we don't have pandas->bigframes logic in two places.

Reply:
Good point! Though I think it's not very straightforward to do so. This is mainly because this function returns a Generator, while the one in the `core` package returns a single value. We will need some extra effort to make everything consistent (function names, parameter types, return types, etc). 

Considering that this RP is already not trivial, I think we can for now only focus on the feature delivery. I will migrate the conversion logic in another PR. Does it sound good to you?

Reply:
b/373716095 for reference